src/Network/Send/kRO/RagexeRE_2018_04_04b.pm

```diff
@@ -18,7 +18,34 @@ use base qw(Network::Send::kRO::RagexeRE_2018_03_07b);
 sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
-	
+
```

Comment:
i have no idea with it .

Reply:
shuffle packets stop at this servertype, so we need to redefine these packets again